packages/ndla-ui/src/MyNdla/CourseCard.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import styled from '@emotion/styled';
+import React from 'react';
+import { fonts, colors } from '@ndla/core';
+import SafeLink from '@ndla/safelink';
+import { IconButtonDualStates } from '@ndla/button';
+import { Heart, HeartOutline } from '@ndla/icons/action';
+import { useTranslation } from 'react-i18next';
+
+const StyledImage = styled.img`
+  object-fit: contain;
+`;
+
+const StyledImageWrapper = styled.div`
+  overflow: hidden;
+  height: 70%;
+  position: relative;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+`;
+
+const CourseTitleWrapper = styled.div`
+  overflow: hidden;
+  height: 30%;
+  position: relative;
+`;
+
+const CourseTitle = styled.h3`
+  justify-content: center;
+  display: flex;
+  align-items: center;
+  color: ${colors.brand.primary};
+  margin: 0;
+  font-weight: ${fonts.weight.semibold};
+  ${fonts.sizes('16')};
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+const LinkWrapper = styled(SafeLink)`
```

Comment:
Bruk flex eller grid på ytterste element (LinkWrapper) i komponenten for å definere layout og eventuelt størrelser på områder.